tests/Model/Asset/AssetThumbnailCacheTest.php

```diff
@@ -89,7 +89,7 @@ public function testThumbnailCache(): void
 
         //load asset via public service controller
         $controller = new PublicServicesController();
-        $subRequest = new Request([
+        $subRequest = new Request([], [], [
```

Comment:
You could write it like this to make it more clear:

```suggestion
        $subRequest = new Request(attributes: [
```